MFBNavigationTests/MFBPushPopNavigatorSpec.m

```diff
@@ -716,6 +716,35 @@
     });
 });
 
+describe(@"external navigation", ^{
+    __block id<UINavigationControllerDelegate> internalDelegate;
+    __block id viewControllerStub;
+    __block id transitionQueueMock;
+
+    beforeEach(^{
+        viewControllerStub = [NSObject new];
+        transitionQueueMock = OCMStrictClassMock([MFBSuspendibleUIQueue class]);
+
+        OCMStub([navigationControllerMock setDelegate:[OCMArg checkWithBlock:^(id obj) {
+            internalDelegate = obj;
+
+            return YES;
+        }]]);
+
+        pushPopNavigator = [[MFBPushPopNavigator alloc] initWithNavigationController:navigationControllerMock
+                                                                     transitionQueue:transitionQueueMock
+                                                                      modalNavigator:modalNavigatorMock];
+
+        OCMStub([navigationControllerMock transitionCoordinator]).andReturn(OCMProtocolMock(@protocol(UIViewControllerTransitionCoordinator)));
```

Comment:
Is it really required?

Reply:
Either this or make navigationControllerMock nice

Reply:
It can still safely return `nil` though.